### PR TITLE
Checkstyle support

### PR DESCRIPTION
**Pull request by mxpv**
Adds checkstyle output format #93

### CODE REVIEW

File: pkg/printers/checkstyle.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			Line:     issue.Line(),
+			Message:  issue.Text,
+			Source:   issue.FromLinter,
+			Severity: defaultSeverity,
+		}
+
+		file.Errors = append(file.Errors, newError)
+	}
+
+	out.Files = make([]*checkstyleFile, len(files))
```
Comment (jirfag) [inline pkg/printers/checkstyle.go]: maybe `make([]*checkstyleFile, 0, len(files))`?

**Comment by jirfag** [comment]
thank you for pull request!

File: pkg/printers/checkstyle.go (hunk 1 above)
Comment (mxpv) [reply pkg/printers/checkstyle.go]: This kind of syntax is not supported for maps.
It'll give an error `too many arguments to make(map)`
https://play.golang.org/p/b7b5e8jgEVp

Comment (jirfag) [reply pkg/printers/checkstyle.go]: But it’s a slice, not map

Comment (mxpv) [reply pkg/printers/checkstyle.go]: oh, crap. yes, you're right :)

**Pull request merged**